components/Feature/TopicExplorer/index.js

```diff
@@ -4,13 +4,17 @@ import Markdown from 'markdown-to-jsx';
 const TopicExplorer = (props) => {
   const [searchTerm, setSearchTerm] = useState('')
   const [searchResults, setSearchResults] = useState([])
+  
+  String.prototype.capitalize = function() {
+    return this.charAt(0).toUpperCase() + this.slice(1).toLocaleLowerCase();
+  }
 
   const getSearchResults = (searchTerm) => {
     var results = []
-    const newSearchTerm = searchTerm.toLowerCase();
+    const newSearchTerm = searchTerm.capitalize();
```

Comment:
Im not sure this is the correct solution because it depends on the tags from airtable being capitalised (which they currently are). An alternative of looping though each tag and altering each of those seems like slow solution. Would like some feedback on this

Reply:
We could remove the first capital letter from the tags in AirTable? So that searches are just lower case?

Reply:
I think we'll find this breaks sometimes - I like lowercasing everything when I'm doing matching like this.

This could be us re-tagging things in Airtable, but I would bet that it'll end up with an wrongly-cased tag at some point, and it'll be opaque to users why things aren't showing up.

I reckon looping through the tags incoming from Airtable might be surprisingly fast - how hard is it to try?

Reply:
@AndyRCarr I think Kyle is right  it'll end up with an wrongly-cased tag so I have now changed it so everything is converted to lowercase.

Reply:
I've now made that change